.circleci/config.yml

```diff
@@ -42,8 +44,8 @@ jobs:
           name: Update golang
           command: |
             sudo rm -rf /usr/local/go/
-            curl -LO https://dl.google.com/go/go1.15.11.linux-amd64.tar.gz
-            sudo tar -C /usr/local -xzf go1.15.11.linux-amd64.tar.gz
+            curl -LO https://go.dev/dl/go1.16.10.linux-amd64.tar.gz
+            sudo tar -C /usr/local -xzf go1.16.10.linux-amd64.tar.gz
```

Comment:
There is also the setup go action: https://github.com/actions/setup-go

Reply:
That is circleCi ;)

Reply:
oh damn :joy: